website/docs/getting-started/for-designers/partials/best-practices.md

```diff
@@ -1,87 +1,101 @@
 
-When design and building local components, consider [Atomic Design](https://bradfrost.com/blog/post/atomic-web-design/) methods whenever possible. Helios components generally correspond with the `atom` and `molecule` levels, with some more complex components being considered `organisms`.
+### Detaching components
 
-## Component instances & detaching
+Our Figma components are coupled tightly with their code counterparts to maintain consistency in the API and design language, so in most scenarios, we don’t recommend detaching a component or foundation from the library.
 
-A component inserted from a library into a project is referred to as an **instance**. Instances maintain a link to the main component in the library, which has significant benefits for deploying updates and iterating quickly on components.
+!!! Info
 
-Our Figma components are coupled closely with their code counterparts to maintain consistency in the API and design language. In most scenarios, we don’t recommend detaching a component or foundation from the library. If you find that a component doesn’t meet your needs, reach out to us for [support](/support) or [submit a request](https://docs.google.com/forms/d/e/1FAIpQLScpMXgrUTVT5fYriu4Pp48r4Nl_eCPluVnJLg0Yg3NXsRWvIA/viewform) for a new component, foundation, or feature.
+**Why to avoid detaching components**
 
-### Why to avoid detaching components
+When you detach a component, it no longer receives updates from the library. This can cause designs and production applications to drift out of sync quickly.
+!!!
 
-When you detach a component, that component will no longer receive updates when new features or changes are published in the library. This can cause Figma projects and production applications to drift out of sync quickly and introduce tech debt in the future. This can be avoided by working closely with the Design Systems Team and learning how to extend components as necessary to meet your needs.
+If you find that a component doesn’t meet your needs, reach out to us for [support](/support) or [submit a request](https://docs.google.com/forms/d/e/1FAIpQLScpMXgrUTVT5fYriu4Pp48r4Nl_eCPluVnJLg0Yg3NXsRWvIA/viewform) for a new component, foundation, or feature.
 
 ### Overriding component styles
 
-While styles in Helios Figma components can technically be overwritten without detaching the component, we recommend against doing this prior to consulting the Design System Team. It may seem simple to change the color or the font size within a Figma component, but this can have drastic trickle-down effects in code.
+While styles in Helios Figma components can technically be overwritten without detaching the component, we recommend against doing this prior to consulting the Design System Team. It may seem simple to change the color or the font size within a Figma component, but this can have trickle-down effects in code.
```

Comment:
```suggestion
While styles in Helios Figma components can technically be overwritten without detaching the component, we recommend against doing this prior to consulting the Design Systems Team. It may seem simple to change the color or the font size within a Figma component, but this can have trickle-down effects in code.
```